conf/messages

```diff
@@ -52,6 +52,7 @@ site.add.item = Add item
 site.add.anotherItem = Add another item
 site.add.country = Add country
 site.search.for.country = Search for country
+site.search.for.location = Search for location
```

Comment:
Could you please add the Welsh for this if available? I approved the PR in case it's to be added later.

Reply:
A translation for this has been requested :)